.github/workflows/check-for-build-warnings.yml

```
@@ -10,6 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
         statuses: write
+        issues: write
```

Should this be pull request:

```suggestion
        pull-requests: write
```